src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public static void RegistrationNavigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
+
+        driver.quit();
+
+    }
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221781259-25f0dbf3-7335-48c9-8ffa-36ce44f19aac.png)

Reply:
I'm unable to follow what the problem is, please help.